internal/commands/commands.go

```diff
@@ -177,4 +178,19 @@ var (
 			},
 		},
 	}
+
+	AccessList = cli.CommandDefinition{
+		CommandMeta: cli.CommandMeta{
+			Use:         "accessList",
+			Aliases:     []string{"accesslist", "access-list"},
+			Description: "Manage your allowed IP addresses and CIDR blocks",
```

Comment:
going to bump this comment one more time, it was resolved but nothing changed...  make this `Description`:

```go
Description: "Manage the allowed IP addresses and CIDR blocks of your Realm app",
```